.github/ISSUE_TEMPLATE/1_activation.md

```diff
@@ -17,34 +17,45 @@ assignees: ''
       * https://api.flutter.dev/
       * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
 
+     Issues directly asking for support will be closed.
+
      If you have found a bug or if our documentation doesn't have an answer
      to what you're looking for, then fill out the template below. Please read
      our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
-## Steps to Reproduce
 
-<!-- Please tell us exactly how to reproduce the problem you are running into. -->
+<!--
+     Please describe in detail the problem you are running into.
+
+     If you have concrete steps to reproduce the problem, list them:
+
+        1. ...
+        2. ...
+        3. ...
+-->
```

Comment:
does this run the risk of people filling in the "..." parts not realising it's commented out?

Reply:
it does, will bring it back out of comment